nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -640,6 +674,16 @@ in {
         ++ (optional (versionOlder cfg.package.version "23") (upgradeWarning 22 "22.05"))
         ++ (optional (versionOlder cfg.package.version "24") (upgradeWarning 23 "22.05"))
         ++ (optional (versionOlder cfg.package.version "25") (upgradeWarning 24 "22.11"))
+        ++ (optional cfg.enableBrokenCiphersForSSE ''
+          You enabled OpenSSL 1.x for NextCloud, this is not necessary except if you
+          are running a RC4-based server side encryption, we advise you to disable server-side
+          encryption and disable this option to run using OpenSSL 3.x.
+
+          See <https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/encryption_configuration.html#disabling-encryption>
+          on how to disable server-side encryption.
+
+          If you think this warning is unwarranted, please open an issue in nixpkgs.
```

Comment:
IMHO the last sentence can be left out.

Reply:
IMHO, I see this pattern for warning that can become pestering & annoying and I do not want to give the impression we explored the whole problem space if we made a mistake and anger people, so I think it's cheap to ask for people to come explain for their use case, but that's my 2 cents. (at least, this is what I would wish for.)

Reply:
The thing is, I'm actually sure you had a different intention (as you stated above), but the wording kind of reminded me of this "X is bad, change my mind" meme. What I want to say, I do think that it people can misunderstand the tone a bit.

But yeah, you do have a point here: how about linking this PR in the warning for more context? If people disagree, they are directly linked to the discussion that lead to this option (to get the full context) and can chime in if they want to.

Reply:
Agreed!